bgpd/bgp_addpath.h

```diff
@@ -15,6 +15,12 @@
 #include "bgpd/bgp_table.h"
 #include "lib/json.h"
 
+struct bgp_addpath_capability {
+	uint16_t afi;
```

Comment:
should this be iana_afi_t?

Reply:
It's used on the receiving side, and since afi is encoded as 2 bytes, iana_afi_t is an integer (not uint16_t), it can't be typecasted correctly.